phivk/gradu.al
# Feature/login with slack

Adds user level auth that can be applied to individual pages as required.

To test: 
- Confirm that https://gradual-git-feature-login-with-slack.phivk.vercel.app/storytellersunited now doesn’t have auth
- Confirm thathttps://gradual-git-feature-login-with-slack.phivk.vercel.app/login requires you to login
- Confirm that after successful login you are redirected to the index page.
